packages/allure-js-commons/src/writers/InMemoryAllureWriter.ts

```diff
@@ -2,6 +2,12 @@ import { PathLike, readFileSync } from "fs";
 import { Category, TestResult, TestResultContainer } from "../model";
 import { AllureWriter } from "./AllureWriter";
 
+export interface AllureResults {
```

Comment:
What happened here? How did this work before

Reply:
that is additional interface, that can be used in together with `InMemoryAllureWriter`. This is part of the future changes I partially picked from other branch (and this interface is added by mistake actually). The idea is to encapsulate writer (return `AllureResults`  instead of `InMemoryAllureWriter`):

```js
describe("Allure XYZ integration tests", () => {
  it("should set status", () => {
    const results: AllureResults = runXyzSpec("testdata/status_spec.ts"); 
    const [ testResult ] = results;
    expect(testResult.status).to.be.eq(Status.PASSED)
  })
})

// xyz_test_utils.ts
const runXyzSpec = (specPath: PathLike): AllureResults => {
   const writer = new InMemoryResultsWriter();
   const reporter = new AllureXyz(writer);
  
   XYZ.runTests(specPath, { reporter });

   return { tests: writer.tests, groups: writer.groups, attachments: writer.attachments };
}

```

Reply:
I'm not feel like sending separate merge request with this change, so I'll merge it

Reply:
ok, as you wish